test/pytest/service-desktop/test_security.py

```diff
@@ -1,33 +1,73 @@
 # Copyright (c) 2017-2021, Mudita Sp. z.o.o. All rights reserved.
 # For licensing, see https://github.com/mudita/MuditaOS/LICENSE.md
+import logging
 
 import pytest
 import time
 
 from harness.interface.defs import status
-
+from harness.api.developermode import PhoneModeLock
+from harness.api.security import GetPhoneLockStatus, GetPhoneLockTime, SetPhoneLockOff
+from harness.request import TransactionError
+
+
+class SecurityTester:
+    def __init__(self, harness):
+        self.harness = harness
+
+    def assert_phone_unlocked(self):
+        assert GetPhoneLockStatus().run(self.harness).response.status == status["NoContent"]
+
+    def assert_phone_locked(self):
+        try:
+            GetPhoneLockStatus().run(self.harness).response.status
+        except TransactionError as error:
+            assert error.status == status["Forbidden"]
```

Comment:
consider logging the error explaining why does it happen

Reply:
Actually, this is the expected status.
Just need to catch it here because some external class raises an exception on any status but _OK_

Reply:
but we assert it here. so it's an expected failure? :)

Reply:
please double check and close this comment :manbow:

Reply:
Yes.
It is a failure just from the perspective of the external class, which raises an exception if the status is different than `OK`.
Here we expect the status to be `Forbidden`, since the phone should be locked when we use this method.
See updated code, now assertion is moved to test. It should be clear now.